.github/workflows/xahau-ga-nix.yml

```diff
@@ -280,6 +291,14 @@ jobs:
           echo "---- Full Environment ----"
           env
 
+      - name: Get commit message
+        id: get-commit-message
+        uses: ./.github/actions/xahau-ga-get-commit-message
```

Comment:
This sets XAHAU_GA_COMMIT_MSG so that [ci-ga-cmake-verbose]

It uses python so this had to be moved down